src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -37,7 +37,7 @@ private fun extractMemberName(part: String): String {
 internal class CodeWriter constructor(
     out: Appendable,
     private val indent: String = "  ",
-    private val memberImports: Set<String> = emptySet(),
+    private val memberImports: Map<String, String?> = emptyMap(),
```

Comment:
should this be `String?` with null as the sentinel? Or say, a special string as we do with `NO_PACKAGE` ?